routing/missioncontrol.go

```diff
@@ -126,6 +126,17 @@ type MissionControlChannelSnapshot struct {
 	SuccessProb float64
 }
 
+// paymentResult is the information that becomes available when a payment
```

Comment:
Commit title+body are pretty bare, just mentions payment results just then makes independent changes to other methods.

Reply:
Commit split in two